requirements-dev.txt

```diff
@@ -15,22 +15,23 @@
 # limitations under the License.
 #
 black==19.10b0
+cachelib==0.1
```

Comment:
Shouldn’t this be defined in `setup.py`?

Reply:
You're probably right, I guess we should consider this an explicit requirement.